source/unified-test-format/schema-latest.json

```diff
@@ -369,6 +378,28 @@
         "connection"
       ]
     },
+    "spanComponent": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "tags"
+      ],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "tags": {
+          "type": "object"
+        },
+        "nested": {
+          "type": "array",
+          "items": {
```

Comment:
Do you intend for the `nested` array to require a minimum of one item (similar to `expectTracingMessages.spans`), or is there a use case for using an empty array here?

I'll note that `minItems` introduces extra overhead for "invalid" tests, so any change here might warrant some additional test files.